FerramAerospaceResearch/FerramAerospaceResearch.csproj

```diff
@@ -21,7 +21,7 @@
     <DebugType>portable</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>$(SolutionDir)bin\$(Configuration)\</OutputPath>
-    <DefineConstants>DEBUG;TRACE;ASSERT;LOG_TRACE</DefineConstants>
+    <DefineConstants>DEBUG;TRACE;ASSERT;LOG_TRACE;ENABLE_PROFILER</DefineConstants>
```

Comment:
Could you add a new configuration for profiling? Not every debug build run needs a profiler

Reply:
Implemented in ab04a71